lib/criteo_api.js

```diff
@@ -950,6 +950,275 @@ class Criteo_API_Client extends API_Client {
         });
     }
 
+    /**
+     * Get the accounts
+     * @param {function} [callback] - Optional callback
+     */
+    getAccounts(callback){
+        return this.criteoApiRequest({
+            'method': 'get',
+            'path': `/preview/retail-media/accounts`,
+            'handler': this.processJSON.bind(this),
+            'callback': callback
+        });
+    }
+
+    /**
+     * Get catalogs by accountId
+     * @param {integer} accountId
+     */
+    getCatalogsByAccountId(accountId, callback){
+        return this.criteoApiRequest({
+            'method': 'post',
+            'path': `/preview/retail-media/accounts/${accountId}/catalogs`,
+            'body': JSON.stringify({}),
+            'headers': {
+                'Content-Type': 'application/json'
+            },
+            'handler': this.processJSON.bind(this),
+            'callback': callback
+        });
+    }
+
+    /**
+     * Get the catalog output
+     * @param {string} catalogId
+     * @param {function} [callback] - Optional callback
+     */
+    //TODO: have to make this just return the string for JSON streaming
```

Comment:
If this is our TODO, it should have a ticket in the backlog.

Reply:
oh yeah, that is supposed to be a comment more than a TODO.... i changed it.